client/scss/layouts/_login.scss

```diff
@@ -23,32 +21,43 @@ $desktop-nice-padding: 100px;
   }
 
   h1 {
-    font-weight: 300;
-    font-size: 2em;
-    line-height: 1em;
-    color: $color-white;
+    color: theme('colors.primary.DEFAULT');
     white-space: nowrap;
```

Comment:
I know this style was here but I recommend against this - it might cause overflow issues for longer translations.